### PR TITLE
[automation] ActionInputsHelper: Enable seconds for time & datetime

### PR DESCRIPTION
**Pull request by florian-h05**
By setting the step size to 1 for time and datetime, seconds are enabled in the UI.

Refs https://github.com/openhab/openhab-webui/pull/2848.
Discussion in https://github.com/openhab/openhab-webui/issues/2847#issuecomment-2446426478

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]

**Comment by florian-h05** [comment]
Once this is merged I will improve Instant and ZonedDateTime support to have the UI display it's dialog and not only a text box.

**Comment by florian-h05** [comment]
@openhab/core-maintainers 
Can you please have a look at these 15 changed lines (where 6 are unit tests) as well as #4438 so we can finish work on the Thing actions before the next milestone and test it in the snapshots before the milestone?

File: bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/util/ActionInputsHelper.java
Comment (J-N-K) [inline bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/util/ActionInputsHelper.java:154]: what does one in this context mean? one second?

Comment (florian-h05) [reply bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/util/ActionInputsHelper.java:154]: Exactly.
The step size for datetime and time contexts defaults to 60 seconds, setting it to 1 second enables them to display seconds as well.

**Review by J-N-K — APPROVED** [review]

**Pull request merged**
